Automate Facebook bot/script.py

```diff
@@ -0,0 +1,51 @@
+import pyautogui
+import time
+import webbrowser
+from selenium import webdriver
+from time import sleep
+from webdriver_manager.chrome import ChromeDriverManager
+from getpass import getpass
+
+usr=input('Enter Email Id:') 
+pwd= getpass('Enter Password:') 
+num = str (input ("Enter comma separated integers: "))
+lists = num.split (",")
+groupid = []
+for i in lists:
+	groupid.append(i)
+
+message=input("Enter your message: ")
+  
+driver = webdriver.Chrome(ChromeDriverManager().install())
+driver.get('https://www.facebook.com/')
+
+  
+username_box = driver.find_element_by_id('email')
+username_box.send_keys(usr)
+
+  
+password_box = driver.find_element_by_id('pass')
+password_box.send_keys(pwd)
+
+  
+login_box = driver.find_element_by_id('u_0_d')
```

Comment:
The problem is that the submit button id is changing at every run. I ran it two times and the ids were different. @antrikshmisri @santushtisharma10 do you have any suggestion to tackle this problem?